src/deepsparse/pipeline.py

```diff
@@ -666,6 +692,20 @@ def to_config(self) -> "PipelineConfig":
             kwargs=kwargs,
         )
 
+    def log(self, identifier: str, value: Any, category: str):
```

Comment:
maybe private method?

Reply:
I was pondering whether to go down that road. Feels right to make it private, but then felt a bit odd given that so many other functions in `pipeline`, that are practically private, do not start with underscore. So I opted for consistency here.

Reply:
subclasses should avoid not calling into private methods when possible, especially if outside the parent module scope, don't see any further reason to hide it as well